cloud/examples/clone_example.cc

```diff
@@ -64,8 +64,9 @@ Status CloneDB(const std::string& clone_name, const std::string& src_bucket,
   // No persistent cache
   std::string persistent_cache = "";
 
+  const std::string bucketPrefix = "rockset.";
   // create a bucket name for debugging purposes
-  const std::string bucketName = "rockset." + kBucketSuffix;
+  const std::string bucketName = bucketPrefix + kBucketSuffix;
```

Comment:
Shouldn't this be cloud_env->GetSrcBucketName()?  As written probably works but assumes both places are changed together.

Reply:
Thanks, changed with https://github.com/thatsafunnyname/rocksdb/commit/bfa8b7fe343e2230d9a5cae1c9b86f6e31405c87